.travis.yml

```diff
@@ -1,6 +1,10 @@
 language: elixir
 elixir:
   - 1.4.1
+cache:
+  directories:
+    - _build
```

Comment:
Don't we want the buid folder to get refreshed at each PR as it will contains the new code? I need to read a bit more to understand exactly how the cache work with Travis.